pyqir/src/builder.rs

```diff
@@ -359,6 +359,36 @@ impl Builder {
         };
         unsafe { Value::from_raw(py, owner, value) }
     }
+
+    /// The ‘zext’ instruction zero extends its operand to the given type.
+    ///
+    /// :param Value val: Value to be converted.
+    /// :param Type ty: Target type.
+    /// :returns: The result.
+    /// :rtype: Value
+    #[pyo3(text_signature = "(self, val, ty)")]
+    fn zext(&self, py: Python, val: &Value, ty: &Type) -> PyResult<PyObject> {
```

Comment:
Corresponding calls need to be added to `pyqir/pyqir/_native.py` showing the python signature and docs. This is needed as python tooling can't pull the native docs when using the language service, so we have to duplicate the docs.

Reply:
Thank you! Done in [b57284a](https://github.com/qir-alliance/pyqir/pull/291/commits/b57284a59fa3e4e96a2f10a5a939e1c1b6a6902c)